Pipfile

```diff
@@ -55,6 +55,7 @@ python-dotenv = "*"
 importlib-resources = "*"
 dictdiffer = "*"
 setuptools = "*"
+responses = "*"
```

Comment:
can't the mocking be done with `requests_mock`? We rather avoid adding dependencies 

also, if we realize it's necessary, please update `Pipfile.lock` as well, and add it to the `dev-packages` part of this file instead (a couple lines below)

Reply:
@dorschw I didn't find a way to return what I sent to a mocked URL with the POST method using `requests_mock`
Is this possible with `requests_mock`?